src/OpenTK.GraphicsLibraryFramework/Enums/ClientApi.cs

```diff
@@ -0,0 +1,33 @@
+//
+// ClientApi.cs
+//
+// Copyright (C) 2019 OpenTK
+//
+// This software may be modified and distributed under the terms
+// of the MIT license. See the LICENSE file for details.
+//
+
+namespace OpenToolkit.GraphicsLibraryFramework
+{
+    /// <summary>
+    /// The context client APIs.
+    /// </summary>
+    /// <seealso cref="IGLFW.WindowHint(WindowHintClientApi,ClientApi)"/>
+    public enum ClientApi
```

Comment:
Consider adding Vulkan to be future-proof.

Reply:
Afaik this is solely used by GLFW and uses encapsulated WGL/GLX/... For OGL context creation. As Vulkan can create it's context with it's own functions there is no need for a specific ClientAPI and would only further complicate things. Therefore we tell GLFW to not create any Context at all(NoApi) die a and leave the Vulkan context creation to the future Vulkan Implementation...

Reply:
Yeah, GLFW can only create an OpenGL context on its own, for a Vulkan context you create one manually, specify NoApi, and then link it to the GLFW window.